lib/internal/cluster/master.js

```diff
@@ -70,6 +79,15 @@ cluster.setupMaster = function(options) {
   assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
          `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
 
+  const hasDebugArg = process.execArgv.some((argv) => {
```

Comment:
@refack the current code does support this use-case, but if the regex is removed as you suggest, and replaced with inspector.url(), then it will stop supporting the use-case.